### DISCUSSION TITLE
Role drop-down is not visible for collaborator on github

### DISCUSSION THREAD

**Discussion by MuhammadUmair786** [community/community#43470] [Repositories] [answered]
### Select Topic Area

Question

### Body

I take this screenshot from [github-guide](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository). 

I created a private repo. But this dropdown is not visible to me. How can I make this visible?

![image](https://user-images.githubusercontent.com/64140593/210998675-0e90ab47-8a73-4e8e-9a73-88403c40eb2a.png)

**Comment by hawkeye359** [comment, marked as answer]
Please refer to this [post](https://stackoverflow.com/questions/64478636/how-to-give-admin-access-to-private-repository-when-there-is-no-role-options).

**Reply by MuhammadUmair786** [reply]
Thank You

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MuhammadUmair786 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
